examples/evaluation/evaluate_with_qa_metrics.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+import openai
+import pandas as pd
+
+import mlflow
+
+logging.getLogger("mlflow").setLevel(logging.ERROR)
```

Comment:
Can we remove this in all the examples since we usually give warnings for errors like toxicity failed because of XYZ and not error messages. people might be confused if they only see errors